app/views/pages/dashboard/_balance_sheet.html.erb

```diff
@@ -147,3 +153,10 @@
     </div>
   <% end %>
 </div>
+
+<%# Custom style for hiding ruler when details are open %>
+<style>
+  details[open] + .group-ruler {
+    display: none;
+  }
+</style>
```

Comment:
If there's a better way to do this let me know! I could add a stimulus but this felt a little cleaner. This is kind of a strange issue - the border being present when the group is open leads to a border on the bottom of the new color. This isn't awful in itself, but it ends up taking up space between two open groups which leads to a white line: 

![CleanShot-003164 2025-05-21 at 14 51 05@2x](https://github.com/user-attachments/assets/ee295afb-11d3-4807-bfea-aa88d0abb58d)

Reply:
Yeah that's a tricky one.  I like your solution a lot better than adding Stimulus to the mix.